src/components/sidebar/SidebarNavigation/index.tsx

```diff
@@ -23,25 +24,29 @@ const Navigation = (): ReactElement => {
   const router = useRouter()
   const { safe } = useSafeInfo()
   const currentSubdirectory = getSubdirectory(router.pathname)
-  const hasQueuedTxs = Boolean(useTxQueue().page?.results.length)
-  const hasRecoveryTxs = Boolean(useRecoveryQueue().length)
+  const queueSize = (useTxQueue().page?.results.length || 0) + useRecoveryQueue().length
```

Comment:
This seems to end up as double the number of actual items in the queue

Reply:
Good catch, fixed!